packages/rsocket-core/src/RSocketClient.js

```diff
@@ -41,6 +41,7 @@ export type ClientConfig<D, M> = {|
     lifetime: number,
     metadata?: string,
     metadataMimeType: string,
+    payload?: string,
```

Comment:
Can we move this to be the first arg so we're consistent? That won't break existing clients right?

Reply:
Sorry for missing in the first round.

Reply:
Moved it to the top, and renamed payload to data. Is this what you meant?

Reply:
And no, it won't break anything as it's an unordered hash.